src/pages/en/guides/markdown-content.md

```diff
@@ -32,7 +32,7 @@ This Markdown file creates a page at `your-domain.com/page-1/`
 
 📚 Read more about Astro's [file-based routing](/en/core-concepts/routing/) or options for creating [dynamic routes](/en/core-concepts/routing/#dynamic-routes).
 
-### Draft Pages
+### Markdown Drafts
```

Comment:
I did think about this! But, we don't have "draft" Astro pages. The only context in which this exists is Markdown/MDX pages... so again, I went back and forth on this one... because, yeah.  😅